pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: vector_tile_renderer
 description: A vector tile renderer for use in creating map tile images or writing to a canvas.
-version: 3.2.1
+version: 3.2.2
```

Comment:
Patch bump since there is no public API or functionality change; just bug fixes.